### ISSUE TITLE
Adding Blazorise.Charts results in TypeLoadException in WASM project

### ISSUE THREAD

**Issue by Krumelur** [stsrki/Blazorise#1135]
**Describe the bug**
I'm following the steps to use the charts extension, as described here: https://blazorise.com/docs/extensions/chart/

**To Reproduce**
1. Add Blazorise.Charts to a Blazor WASM project (`dotnet add package Blazorise.Charts`)
2. Build the app
3. Try to debug the app (Edge and Chrome)
4. See error:

```
System.TypeLoadException: Type Blazorise.Bootstrap.BootstrapClassProvider has invalid vtable method slot 11 with method Blazorise.ClassProvider:SelectMultiple ()
localhost꞉5001/_framework/blazor.webassembly.js:1
  at DashboardClient.Program.Main (System.String[] args) [0x00033] in /Users/rene/Develop/Dashboard/src/Client/Program.cs:22 
localhost꞉5001/_framework/blazor.webassembly.js:1
  at Microsoft.AspNetCore.Components.WebAssembly.Hosting.EntrypointInvoker.InvokeEntrypoint (System.String assemblyName, System.String[] args) <0x26ee428 + 0x0012c> in <filename unknown>:0 
```

It's worth noting that nothing else has been done. Just adding the packages generates the error.

**Expected behavior**
No error :-)

https://github.com/Krumelur/Dashboard/tree/charts/src/Client

**Comment by stsrki** [comment]
I think you're missing javascript files in `index.html`

```
<script src="https://cdn.jsdelivr.net/npm/chart.js@2.8.0"></script>
<script src="_content/Blazorise.Charts/blazorise.charts.js"></script>
```

https://blazorise.com/docs/extensions/chart/#index

**Comment by Krumelur** [comment]
Same thing when I add them. I followed all of the instructions and got the error. 
Then I started removing things and the error persisted until I also removed the Nuget package. 

- Add the Nuget to the project => error
- Remove it => all good

I'll try and create a self contained demo app for further investigation.

**Comment by Krumelur** [comment]
Closing this. I created a new test project and it works, so it must be something in my project. Maybe some package dependency issue.

**Comment by stsrki** [comment]
Maybe TargetFramework was not supported. Blazorise is using `.netstandard2.1`

**Comment by Krumelur** [comment]
Dunno...I got past the error. But now I'm seeing `Unhandled exception rendering component: Could not find 'blazoriseCharts' in 'window'.`. However, this ONLY happens when running the Blazor projects from within a Visual Studio Code Workspace that contains the Blazor project. It's fine when opening the folder directly and running it from there.

**Comment by WillianGruber** [comment]
10% of the times, when I execute my project from vs with ctrl+f5, I get this error "Could not find blazorise in window". Never investigated it, though.

**Comment by stsrki** [comment]
I never used Blazor with VS Code. Don't know how the situations is now but previously support was not good. I think best is for you to use Visual Studio if you can.

**Comment by mcguireuk** [comment]
I know you have closed this one but I've just seen this same error. I updated Blazorise to latest version and started getting this error. Turns out I forgot to update the other Blazorise packages I was using to the same version. This fixed the issue. I'm guessing when you installed the charts. it was a different version from your other packages. This being fixed in your new test project as they would all be the latest versions.
